### DISCUSSION TITLE
My post was incorrectly merged and closed

### DISCUSSION THREAD

**Discussion by dr-nyt** [community/community#22223] [New to GitHub]
<p>I made a <a href="https://github.community/t/make-the-github-release-button-more-apparent/121751">post</a> yesterday regarding a small change related to the GitHub releases button placement in the new UI.</p>
<p>I am not ranting about the UI, just a rational suggestion, but I think it was incorrectly merged into <a href="https://github.community/t/how-to-opt-out-of-the-new-ui/119416">How to opt-out of the new UI?</a> and closed.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/dr-nyt">@dr-nyt</a>! Thank you for reaching out on this.  I apologize for the misunderstanding; I’ve corrected that.  You should be able to see your previous topic live now.</p>

**Comment by dr-nyt** [comment]
<p>No problem, and thank you.</p>

**Comment by MrMacdonalds** [comment]
<p>your a dr well im having a hard time in life ive been pushed to the streets or homeless at my rents when i have over a mill in the bank. anyway i need help with tech and need the doc to sign off on my license . ill pay you too for your trouble.</p>
